src/Cake.Common/Tools/SignTool/SignToolResolver.cs

```diff
@@ -67,7 +67,9 @@ private FilePath GetFromDisc()
             if (_environment.Platform.Is64Bit)
             {
                 // 64-bit specific paths.
-                files.Add(programFilesPath.Combine(@"Windows Kits\10\bin\x64").CombineWithFilePath("signtool.exe"));
+                files.Add(programFilesPath.Combine(@"Windows Kits10\bin\x64").CombineWithFilePath("signtool.exe"));
+                files.Add(programFilesPath.Combine(@"Windows Kits\10\bin\10.0.15063.0\x64").CombineWithFilePath("signtool.exe"));
+                files.Add(programFilesPath.Combine(@"Windows Kits\10\bin\10.0.14393.0\x64").CombineWithFilePath("signtool.exe"));
```

Comment:
Instead of hardcoding versions (which would make us add additional path everytime a new version of Win10 SDK is released) could we instead search using pattern `Windows Kits/10/bin/*/x64/signtool.exe`?